package.json

```diff
@@ -14,35 +14,37 @@
   "homepage": "https://github.com/FormidableLabs/builder-react-component",
```

Comment:
Update homepage to `https://github.com/FormidableLabs/builder-react-component-dev`

Reply:
Several refs to the repo need updating.

Reply:
Will we need scripts for testing and publishing etc?